src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -235,6 +244,7 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
     const base::ReportIntermediateSolutionFn intermediateSolutionCallback = pdef_->getIntermediateSolutionCallback();
 
     Motion *solution = lastGoalMotion_;
+    Motion *solution_tmp       = lastGoalMotion_;
```

Comment:
FYI, `lastGoalMotion_`, it is a misleading variable. It holds the last path to a goal region, not necessarily the best path. It is really only used for `Planner::getPlannerData`, and I am planning to remove it and just iterate over `goalMotions_` when necessary.